### PR TITLE
Add additional Storepoint spiders

### PR DESCRIPTION
**Pull request by davidhicks**

### CODE REVIEW

File: locations/spiders/digi_telecommunications_my.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+from locations.storefinders.storepoint import StorepointSpider
+
+
+class DiGiTelecommunicationsMYSpider(StorepointSpider):
+    name = "digi_telecommunications_my"
+    item_attributes = {"brand": "DiGi Telecommunications", "brand_wikidata": "Q3268530"}
+    key = "1624a4f7eafca0"
+
+    def parse_item(self, item, location: {}, **kwargs):
+        item["addr_full"] = item.pop("street_address")
```
Comment (Cj-Malone) [inline locations/spiders/digi_telecommunications_my.py:13]: If this is common across all StorepointSpider spiders, it should go in `StorepointSpider`. Same goes for the opening hours.

File: locations/spiders/digi_telecommunications_my.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        item["country"] = "MY"
```
Comment (Cj-Malone) [inline locations/spiders/digi_telecommunications_my.py:14]: This isn't actually necessary, we get it from the spider name now: digi_telecommunications_**my**

File: locations/spiders/digi_telecommunications_my.py (hunk 1 above)
Comment (davidhicks) [reply locations/spiders/digi_telecommunications_my.py:13]: It does indeed look common with street_address and addr_full, as well as the opening hours format. I'll change the Storepoint spider.

**Comment by davidhicks** [comment]
Cleaned up and reorganised the patch set with changes made to address feedback.

File: locations/spiders/digi_telecommunications_my.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+
+class DiGiTelecommunicationsMYSpider(StorepointSpider):
+    name = "digi_telecommunications_my"
+    item_attributes = {"brand": "DiGi Telecommunications", "brand_wikidata": "Q3268530"}
+    key = "1624a4f7eafca0"
+
+    def parse_item(self, item, location: {}, **kwargs):
+        if item["phone"] is not None:
+            item["phone"] = "+60-" + re.sub(r"^601", "1-", re.sub(r"^03-", "3-", item["phone"]))
```
Comment (iandees) [inline locations/spiders/digi_telecommunications_my.py:13]: You shouldn't need to do any phone parsing/manipulation. We normalize the phone number during the scraping process.

Comment (davidhicks) [reply locations/spiders/digi_telecommunications_my.py]: Thanks, I've removed it. I think at one point I was getting errors about some invalid phone numbers and therefore tried to rewrite the numbers manually. I must have confused spiders as it isn't needed for DiGi.

**Pull request merged**